### ISSUE TITLE
[Feature Request] Support for requestBody being required

### ISSUE THREAD

**Issue by bspellacy** [rswag/rswag#341]
# Problem
Open API 3.x adds support for `requestBody` to be required. By default, it is optional. Currently, rswag doesn't have the option to declare the request body as mandatory.

# Proposed Solution
Add support for a `required` key to the `parameter` method. If the parameter is `in: :body`, the `required` boolean will be respected. E.g.
```ruby
parameter name: :order, in: :body, required: true, schema: { '$ref' => '#/components/schemas/order' }
```
I believe modifying this [method](https://github.com/rswag/rswag/blob/bec4f16676c6bd7b5ef14cf0dc340a6525429567/rswag-specs/lib/rswag/specs/swagger_formatter.rb#L142) should be enough to get it done. I'd be happy to open up a PR once the team okays the approach.

**Comment by bspellacy** [comment]
Let me know what you think @BookOfGreg! Shouldn't be too tough to knock this out quickly.

**Comment by bspellacy** [comment]
Addressed in: #342

**Comment by thdaraujo** [comment]
+1 for this!
